packages/amplication-client/src/Project/ProjectSideBar.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Project, Workspace } from "../models";
+import ProjectList from "./ProjectList";
+import "./ProjectSideBar.scss";
+import ProjectSideBarFooter from "./WorkspaceSettingsNavigation";
+import WorkspaceSelectorNew from "./WorkspaceSelectorNew";
+
+const CLASS_NAME = "project-sidebar";
+
+type Props = {
+  currentWorkspace: Workspace;
+  currentProject: Project;
+  projectsList: Project[];
+  setNewProject: (project: Project) => void;
+  newProjectRes: () => void;
+  handleSetCurrentWorkspace: (workspaceId: string) => void;
+};
+
+const ProjectSideBar = ({
+  currentWorkspace,
+  currentProject,
+  projectsList,
+  setNewProject,
+  newProjectRes,
```

Comment:
what is this?

Reply:
These are the props that I suppose to get from the Context